app/containers/MassEnergizeSuperAdmin/Actions/AllActions.js

```diff
@@ -232,15 +238,15 @@ class AllActions extends React.Component {
     }
 
     const options = {
-      filterType: 'dropdown',
-      responsive: 'stacked',
+      filterType: "dropdown",
+      responsive: "stacked",
       print: true,
-      rowsPerPage: 100,
+      rowsPerPage: 15,
```

Comment:
15 rows a page is pretty small.  Would be good if it was 25 or something but we changed it to be 100.

Reply:
I made some pages 15 because loading items that have images on them makes the whole thing lag for a short time when you load on to the page. That is why for every page that we are showing images on the table, I have reduced the number of items to show at once. 

And yes, it was 100 before. I can change it back if you dont think lags are a deal breaker 🤣 . I think they are though.

Reply:
Also, i figured this isnt a problem because everytime an admin smoothly loads on to the page of (25), they could easily just switch the table to show 100 at once right...?


And every time I tried switching manually to 100 this way, the performance was way better, and there was no lag.... 

For many other pages that done have images, I showed more, but just enough to prevent lagging as well. 

With all the computations we are doing to come up with the right display of the tables, its better if we just show a small portion of the data on first load, then when all has settled, the user gets to choose to view more; at which time, not a lot of other processes are going on, so showing more is **waay** smoother.